nmrfx-processor-gui/src/main/java/org/nmrfx/processor/gui/MainApp.java

```diff
@@ -76,6 +78,11 @@ public class MainApp extends Application {
     protected static MainApp mainApp = null;
     static boolean isAnalyst = false;
     static Font defaultFont;
+    // Icon and font sizes for icon buttons
+    public static final String ICON_SIZE_STR = "16px";
+    public static final String ICON_FONT_SIZE_STR = "7pt";
+    // The default font size
+    public static final String REG_FONT_SIZE_STR = "9pt";
```

Comment:
icon and font size for icon buttons can't be set in css files due to GlyphDudes.createIconButton method calls, the regular font size is also set here instead of in css so it may possible be changed programmatically in the future (request from @brucejohnson )